docker/Dockerfile

```diff
@@ -40,25 +29,14 @@ RUN INFER_VERSION=v0.12.1; \
     ln -s ${PWD}/infer-linux64-$INFER_VERSION /infer
 
 # Compile Infer
-RUN OCAML_VERSION=4.04.2; \
-    opam init --compiler=$OCAML_VERSION -j $(getconf _NPROCESSORS_ONLN || echo 1) --yes && \
-    cd /infer && \
-    ./build-infer.sh --opam-switch $OCAML_VERSION && \
-    rm -rf /root/.opam
+RUN OCAML_VERSION=4.05.0; \
+    opam init --compiler=$OCAML_VERSION -j $(nproc) --yes && \
+    cd /infer && ./build-infer.sh --opam-switch $OCAML_VERSION && rm -rf /root/.opam
 
 # Install Infer
 ENV INFER_HOME /infer/infer
 ENV PATH ${INFER_HOME}/bin:${PATH}
 
-# Install dependencies for Android sample
```

Comment:
This used to be installed separately, but I'm not sure if this matters much. @3flex?

Reply:
My thought behind this was that we could do it all in one shot when the initial `apt-get update && apt-get install` is done. Also saves a layer :)